source/includes/_budget.md

```diff
@@ -2,31 +2,43 @@
+exclude_from_budget | boolean | false | If true, this category is excluded from budget (category properties are set in the app via the Categories page)
+exclude_from_totals | boolean | false | If true, this category is excluded from totals (category properties are set in the app via the Categories page)
+data                | Array of data objects | false | For each month with budget or category spending data, there is a data object with the key set to the month in format YYYY-MM-DD. For properties, see Data object below.
+config              | object | true | Object representing the category's budget suggestion configuration
 
 ## Data Object
 
-Attribute Name      | Type   | Description
-------------------- | ----   | -----------
-budget_amount       | number | The budget amount, as set by the user. If empty, no budget has been set.
-budget_currency     | string | The budget currency, as set by the user. If empty, no budget has been set.
-budget_to_base      | number | The budget converted to the user's primary currency. If the multicurrency feature is not being used, budget_to_base and budget_amount will be the same. If empty, no budget has been set.
-spending_to_base    | number | The total amount spent in this category for this time period in the user's primary currency
-num_transactions    | number | Number of transactions that make up "spending_to_base"
-is_automated        | boolean | If true, the budget_amount is only a suggestion based on the set config. If not present, it is false (meaning this is a locked in budget)
+Attribute Name      | Type   | Nullable | Description
+------------------- | ----   | -------- | -----------
+budget_amount       | number | true | The budget amount, as set by the user. If empty, no budget has been set.
+budget_currency     | string | true | The budget currency, as set by the user. If empty, no budget has been set.
+budget_to_base      | number | true | The budget converted to the user's primary currency. If the multicurrency feature is not being used, budget_to_base and budget_amount will be the same. If empty, no budget has been set.
+spending_to_base    | number | false | The total amount spent in this category for this time period in the user's primary currency
+num_transactions    | number | false | Number of transactions that make up "spending_to_base"
+is_automated        | boolean | true | If true, the budget_amount is only a suggestion based on the set config. If not present, it is false (meaning this is a locked in budget)
+
+## Config Object
+
+Attribute Name      | Type   | Nullable | Description
+------------------- | ----   | -------- | -----------
+config_id           | number | false | Unique identifier for config
+cadence             | string | false | One of:<br> <ul> <li>monthly</li> <li>twice a month</li> <li>once a week</li> <li>every 3 months</li> <li>every 3 months</li> <li>every 4 months</li> <li>twice a year</li> <li>yearly</li></ul>
```

Comment:
Based this description on the cadence description for another object and the example